### ISSUE TITLE
Customize button don't update ProductContext correctly  

### ISSUE THREAD

**Issue by gustavopvasconcellos** [vtex-apps/product-customizer#79]
**Describe the bug**

TL;DR: We configurated an assembly options attachment in some products and while we tested native components they didn't work.

This product below have other assembly as an assembly options, this enables the "Customize" button and opens a Modal on click.

https://mcdon144--acctglobal.myvtex.com/mcproductrefid136/p

But the ProductContext is updated with the assembly options of the combo:

![image](https://user-images.githubusercontent.com/49173685/101687387-2c964000-3a49-11eb-8304-38c371ff61d5.png)

And the combo haven't been updated on his assembly option, the correct assembly is "Lanche Generico":

![image](https://user-images.githubusercontent.com/49173685/101688399-bb578c80-3a4a-11eb-8417-ac02d6dc82b5.png)

**To Reproduce**

1. Go to https://mcdon--acctglobal.myvtex.com/mcproductrefid136/p
2. Try to change the options and debugg the add to cart button.

**Expected behavior**
The correct behavior is to change the assembly option items, not to add new assemblies on the main product.
